File: string/reverse-string/reverse-string.go
package reversestring

func reverseString(s []byte) {
	i := 0
	j := len(s) - 1
	for i < j {
		aux := s[i]
		s[i] = s[j]
		s[j] = aux
		i++
		j--
	}
}

File: string/reverse-string/reverse-string.ts

/**
 Do not return anything, modify s in-place instead.
 */
function reverseString(s: string[]): void {
    let i = 0, j = s.length - 1;

    while (i < j) {
        const aux = s[i];
        s[i++] = s[j];
        s[j--] = aux;
    }
};
